src/cmd/peer/ListPeers.ts

```diff
@@ -5,6 +5,7 @@ const listPeer = async (): Promise<void> => {
   Peer.find({}, '-_id -certificate -fingerprint', (error, peers) => {
     printPeers(peers)
   })
+  process.exit(1)
```

Comment:
Looks like I merged too hastily. Should be placed after printPeers() and return `process.exit(0)` to indicate successful run.